dotCMS/src/main/java/com/dotcms/rest/api/v1/maintenance/MaintenanceResource.java

```diff
@@ -73,7 +76,14 @@ public final Response shutdown(@Context final HttpServletRequest request,
             return Response.status(Status.FORBIDDEN).build();
         }
 
-        Runtime.getRuntime().exit(0);
+        DotConcurrentFactory.getInstance()
+                .getSubmitter()
+                .submit(
+                        () -> Runtime.getRuntime().exit(0),
```

Comment:
Codacy found an issue: [System.exit() should not be used in J2EE/JEE apps](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6380610)